### DISCUSSION TITLE
Simple question: What does 'half' mean?

### DISCUSSION THREAD

**Discussion by mikedolx** [dylandoamaral/uptime-card#179] [Q&A] [answered]
Hi,

I briefly searched the README, the issues and discussions, but couldn't find any info about. 
What does the color half mean? I didn't saw this color in my uptime instance (there's only red and green).

Thanks

**Comment by dylandoamaral** [comment, marked as answer]
If you don't user https://github.com/dylandoamaral/uptime-card?tab=readme-ov-file#color-thresholds, the bar can appear in three color: ok, ko, half. To understand why we need three colors, you have to understand that a bar in uptime card can represent a time slot and during this time slot, your device can turn on and off several times. So during this slot your device can be for example 70% of the time ON and 30% of the time OFF. So you have something called severity to tell you how to handle this case. By default it is 100 meaning that the bar will be KO if 100% of the time the device was off, OK if 100% of the time the device was on and half the rest of the time.

**Reply by mikedolx** [reply]
Great answer! Thanks s lot. 

That type of "aggregation" gives a quite interesting view of my homelab. Apparently, I have times in my network, where all services have trouble being connected to.

![markup_29960 (1).png](https://github.com/user-attachments/assets/c0e189ed-bff3-4d3d-950f-21bf191df51c)
